$OpenBSD: patch-data_ui_rhythmbox-ui_xml,v 1.3 2012/09/30 14:58:39 ajacoutot Exp $

audiocd plugin requires cdda support in gvfs which we do not provide as
this would require HAL or gudev.

--- data/ui/rhythmbox-ui.xml.orig	Sun Sep 30 04:03:28 2012
+++ data/ui/rhythmbox-ui.xml	Sun Sep 30 12:30:39 2012
@@ -3,8 +3,6 @@
     <menu name="MusicMenu" action="Music">
       <menuitem name="MusicImport" action="MusicAdd"/>
       <separator/>
-      <menuitem name="MusicCheckDevices" action="MusicCheckDevices"/>
-      <separator/>
       <menu name="PlaylistMenu" action="Playlist">
 	<menuitem name="MusicPlaylistNewPlaylistMenu" action="MusicPlaylistNewPlaylist"/>
 	<menuitem name="MusicPlaylistNewAutomaticPlaylistMenu" action="MusicPlaylistNewAutomaticPlaylist"/>
